### ISSUE TITLE
Whitelist ldap_escape

### ISSUE THREAD

**Issue by aidanlane** [scrutinizer-ci/php-analyzer#618]
Shouldn't ldap_escape as a valid way of escaping ldap strings passed to ldap_search, so that we're not alerted of a critical "1 path for user data to reach this point" issue? Or my reasoning wrong on this?

Thanks,
Aidan

see https://scrutinizer-ci.com/g/aidanlane/alexandria/inspections/6db53c4a-3317-4642-a760-296818bb8925/issues/files/alex-auth/auth.php?status=new&orderField=path&order=asc&issueId=595bd36d-fcb2-4557-b5c7-0b00aa21a687

**Comment by schmittjoh** [comment]
Indeed, this should be recognized by the analyzer, thanks for the report.

**Comment by aidanlane** [comment]
Fantastic, thanks @schmittjoh :)
